src/components/index.js

```diff
@@ -0,0 +1,8 @@
+import ProductCharacteristics from './ProductCharacteristics';
```

Comment:
This file is not covered. Please provide unit test.

Reply:
done